### ISSUE TITLE
Follow button on sidebar has duplicate aria-label

### ISSUE THREAD

**Issue by saihla** [berkmancenter/frankly#74]
The follow buttons on a community page and in the sidebar have the same aria-labels. Should be distinct for recognition by Playwright tests and screen readers
